Troubleshooting.md

```diff
@@ -112,7 +112,11 @@ Cause of error is for now unknown, but something caused vscode java support and
 
 ### Try:
 1. Restart VS Code and the issue should disappear 
-2. If it continues to error try restart again, and if still a problem open an issue at [vscode-java-debug](https://github.com/Microsoft/vscode-java-debug)
+2. If it continues to error try restart again
+3. If the version of Java Debugger extension and RedHat Java extension are not the most recent version, referring [#1342](https://github.com/microsoft/vscode-java-debug/issues/1342), please update both of them to the most recent version, and reboot VS Code
+4. If the version of Java Debugger extension and RedHat Java extension are the most recent version or even the newest, and if using the Test Runner for Java extension, referring [#1308](https://github.com/microsoft/vscode-java-debug/issues/1308), please downgrade the version of Test Runner for Java extension, then

```

Comment:
It doesn't make sense to me, personally.

If we can confirm that Test Runner may cause this issue, we should find the root cause and fix it. This statement does not provide enough information. Which version does user expect to downgrade?

Reply:
+1, all extensions in vscode-java-pack should be compatible with their latest versions.